docs/modules/ROOT/pages/performance_analysis/dpmAcaGrpcTest.adoc

```diff
@@ -0,0 +1,215 @@
+= ALCOR CONTROL AGENT-ALCOR DATAPLANE MANAGER Test Report 
+Xiaodong Zhang <xzhang2@futurewei.com>
+v0.1, 2020-09-15
+:toc: right
+:imagesdir: ../../images
+
+*ALCOR CONTROL AGENT-ALCOR DATAPLANE MANAGER Test Report*
+
+**Abstract：**This document contains Alcor Dataplane Manager-Alcor Control Agent grpc performance test result and analysis
+
+[arabic]
+. *Environment description*
+
+[cols=",",options="header",]
+|===
+|*IP address* |
+|*A* |*10.213.43.162*
+|*B* |*10.213.43.163*
+|*C* |*10.213.43.164*
+|*D* |*10.213.43.166*
+|*E* |*10.213.43.187*
+|F |*10.213.43.188*
+|===
+
+[cols=",,",options="header",]
+|===
+|*Hardware Configuration:* | |
+| |*A, B, C, D, E running ALCOR CONTROL AGENT* |*F running as ALCOR DATAPLANE MANAGER*
+|*CPU* |40 cores |56 cores
+|*Model Name* |Intel(R) Xeon(R) CPU E5-2690 v2 @ 3.00GHz |Intel(R) Xeon(R) CPU E5-2697 v3 @ 2.60GHz
+|*cpu MHz* |2231.772 |2599.079
+|*Memory* |192GB |386GB
+|*Network* |NetXtreme BCM5719 Gigabit Ethernet PCIe (GB network) |82599ES 10-Gigabit SFI/SFP+ Network Connection
+|*Storage* |LSI raid (no ssd) |AVAGO (no ssd)
+|*Software Configuration:* | |
+|*System* |Ubuntu18 LTS |
+|*Server* |ALCOR CONTROL AGENT on A, B, C, D, E |
+|*Client* |ALCOR DATAPLANE MANAGER on F |
+|===
+
+[arabic, start=2]
+. *Test step:*
+
+F send goal state message to A-E at the same time concurrently after first warming up then wait for the response, goal state message is different in each payload
```

Comment:
Can you upload the test scripts or codes that generate the payload to https://github.com/futurewei-cloud/alcor-int/tree/master/tools? This can be done in a sperate PR.